.github/workflows/test-cli.yml

```diff
@@ -0,0 +1,39 @@
+name: CFT CLI Tests
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'cli'
+      - '.github/workflows/**'
+  pull_request:
+    branches:
+      - 'master'
+    paths:
+      - 'cli'
+      - '.github/workflows/**'
+
+concurrency:
+  group: '${{github.workflow}}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  unit:
+    name: CLI unit tests
+    runs-on: ${{ matrix.operating-system }}
+    defaults:
+      run:
+        shell: bash
+        working-directory: 'cli'
+    strategy:
+      fail-fast: false
+      matrix:
+        operating-system: [ubuntu-latest, macos-latest]
```

Comment:
Windows seems to fail some tests. I suspect line-endings, but will test and reenable in a follow up.
https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/runs/4447248143?check_suite_focus=true